docs/runbook.md

```
@@ -335,6 +335,90 @@ mc ilm rule add --expire-days 1 --tags "committed=false" myminio/<your-bucket>
+       count(*) FILTER (WHERE "processedAt" IS NULL)     AS pending
+FROM outbox_events
+GROUP BY 1
+ORDER BY 1 DESC
+LIMIT 14;
+```
+
+If `ENABLE_METRICS=true`, the `outbox_lag_seconds` gauge reflects the age of the oldest unprocessed event in real time — alert when it trends upward after the relay is healthy.
+
+**Escalation:** If processed rows accumulate despite the purge running successfully (i.e. the cron completes but `count(*)` still climbs), the event volume exceeds `OUTBOX_PURGE_MAX_BATCHES × OUTBOX_PURGE_BATCH_SIZE`. Calculate the required cap:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)